skglm/solvers/fista.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+from scipy.sparse import issparse
+from skglm.solvers.base import BaseSolver
+from skglm.solvers.common import construct_grad, construct_grad_sparse
+from skglm.utils import _prox_vec
+
+
+class FISTA(BaseSolver):
+    r"""ISTA solver with Nesterov acceleration (FISTA).
+
+    This solver implements accelerated proximal gradient descent for linear problems.
```

Comment:
What does linear mean here ? If it's that the datafit is quadratic, it'd be better to state it explicitely

Reply:
I think "ISTA solver with Nesterov acceleration (FISTA)." is enough.
We better remove this statement.
WDYT?

Reply:
I agree :+1: